examples/test_tsd_displacement.py

```diff
@@ -0,0 +1,152 @@
+# [THINSHELLDEMONS] Read Mesh and Create TSD object
```

Comment:
I think you need the [license header](https://github.com/InsightSoftwareConsortium/ITKMontage/blob/v0.7.3/examples/SimpleMontage.py#L1-L17) here.